ports/kissfft/vcpkg.json

```
@@ -1,7 +1,17 @@
 {
   "name": "kissfft",
-  "version-string": "2020-03-30",
+  "version-string": "2021-11-14",
```

```suggestion
  "version-date": "2021-11-14",
```
This should be served by `version-date`.

---

Doesn't this change will break as previous version use "version-string" ?

---

Doesn't this change will break version management as previous iteration use version-string